docs/contracts/v2/reference/02-diagrams.md

```diff
@@ -15,26 +15,119 @@ would still be valid.
 
 :::
 
+## Lockup Linear
+
+An example of two streams being stored in the Lockup Linear contract.
+
+```mermaid
+flowchart LR;
+		classDef object stroke:#ff9C00,color:#ffffff;
+    contract["Lockup Linear
+		contract"];
+		stream_1[(Stream LL-1-1)];
+		property_1([Start time]);
+		property_2([End time]);
+		property_3([Deposit amount]);
+		stream_1 --> property_1;
+		stream_1 --> property_2;
+		stream_1 --> property_3;
+		contract --> stream_1;
+
+		stream_2[(Stream LL-1-2)];
+		property_4([Start time]);
+		property_5([Stop time]);
+		property_6([Deposit amount]);
+		stream_2 --> property_4;
+		stream_2 --> property_5;
+		stream_2 --> property_6;
+		contract --> stream_2;
+		class contract,stream_1,stream_2,property_1,property_2,property_3,property_4,property_5,property_6 object;
+```
+
+## Lockup Dynamic
+
+An example of two streams being stored in the Lockup Dynamic contract.
+
+```mermaid
+flowchart LR;
+		classDef object stroke:#ff9C00,color:#ffffff;
+    contract["Lockup Dynamic
+		contract"];
+		stream_1[(Stream LD-1-1)];
+		property_1([Start time]);
+		property_2([End time]);
+		property_3([Deposit amount]);
+		property_4["segment 1
+		segment 2
+		segment 3
+		segment 4
+		...
+		segment n"]
+		stream_1 --> property_1;
+		stream_1 --> property_2;
+		stream_1 --> property_3;
+		stream_1 --> property_4;
+		contract --> stream_1;
+
+		stream_2[(Stream LD-1-2)];
+		property_5([Start time]);
+		property_6([Stop time]);
+		property_7([Deposit amount]);
+		property_8["segment 1
+		segment 2
+		segment 3
+		segment 4
+		...
+		segment n"]
+		stream_2 --> property_5;
+		stream_2 --> property_6;
+		stream_2 --> property_7;
+		stream_2 --> property_8;
+		contract --> stream_2;
+
+		class contract,stream_1,stream_2,property_1,property_2,property_3,property_4,property_5,property_6,property_7,property_8 object;
+```
+
+## Protocol participants
+
+```mermaid
+flowchart LR;
+		classDef object stroke:#ff9C00,color:#ffffff;
+    contract[Sablier V2];
+		developers[Developers]
+		governance[Protocol Admin]
+		users[Stream creators/recipients]
+
+		governance--"governs"-->contract;
+		users--"use"-->contract;
+		developers--"maintain"-->contract;
+
+		class contract,developers,governance,users object;
+```
```

Comment:
I'm sorry; I should have clarified that the intention with this diagram was not to use Mermaid.

Mermaid doesn't have user silhouettes, which are critical in this context of explaining what the protocol participants are.

Reply:
No worries, wasn't really too sure of that one anyway.